src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Resources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ClassTemplateName" xml:space="preserve">
```

Comment:
Is the class template name localized? What would we call Class in other languages?